### PR TITLE
Add leaving and viewing comments function

### PR DESCRIPTION
**Pull request by nepecg000**
Thanks for reviewing. This servlet works under /data URL.

### CODE REVIEW

**Review by layog — COMMENTED** [review]
This contains commits from other PRs. Can you remove those commits?

Also, there is only one new commit in this PR where you add your name after "Hello World". The title of the PR does not match the content here

**Review by layog — CHANGES_REQUESTED** [review]
Hi Hsin-mei,

This PR still contains commits from the other PR, while merging it will give you lots of merge conflicts. So I suggest to fix this first. If you have trouble, I'll explain how to do this in our meeting this time.

Can you deploy this version as well? I cannot see the comments feature on the deployed version.

**Comment by nepecg000** [comment]
I tried to fix the problem by rebasing week2 from week1 to master. However, it seems that every file that week1 creates, but week2 doesn't modify it, was deleted.

**Comment by layog** [comment]
Okay, let's do one thing, merge the PR#4, I have approved that, then pull the master locally and try rebasing to master again.

**Comment by nepecg000** [comment]
This looks more making sense now. Please review it and see if that's right.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     
         <link rel="stylesheet" type="text/css" href="./css/indexStyle.css">
 
+        <script src="./script.js"/>
```
Comment (layog) [inline portfolio/src/main/webapp/index.html:11]: nit: AFAIK we do not prefer relative paths in HTML. I believe you should change it to absolute path like `src="/script.js"`

**Review by layog — APPROVED** [review]
I like the readability of your code, really well written. LGTM

**Pull request merged**